### PR TITLE
Issue 111 integrate dw erection: Merge private (LandBOSSE-dev) ErectionCost module into public ErectionCost module

### PR DESCRIPTION
**Pull request by barker59**
Integrates DW functionality from LandBOSSE-dev
Ready for review.

### CODE REVIEW

**Comment by parangat94** [comment]
Parangat currently reviewing this.

**Comment by parangat94** [comment]
@akey7 @eberlea  ready for review.

File: landbosse/model/ErectionCost.py [hunk 1]
```diff
@@ -135,10 +143,15 @@ class documentation
         output_dict : dict
             The output dictionary with key value pairs as found on the
             output documentation.
+
+        log
+            Python logging service for logging.
```
[the remark this answers is not in the document]
Comment (barker59) [reply landbosse/model/ErectionCost.py]: @akey7 Made the changes requested, let me know if those work for you!

**Comment by akey7** [comment]
@barker59 Please see my email about validation.

**Comment by barker59** [comment]
@akey7 Made requested changes. Removed doubling of mobilization cost for both DW and non-DW projects in ErectionCost.py

**Comment by parangat94** [comment]
@barker59 @akey7 is this ready to be merged?

**Comment by akey7** [comment]
@barker59 @parangat94 Now all the Erection mobilization validation cases, DW and non-DW, are broken. So it is not ready to merge yet.

File: main.py [hunk 2]
```diff
@@ -48,7 +48,7 @@
     extended_project_list_path = os.path.join(file_ops.extended_project_list_path(), 'extended_project_list.csv')
     extended_project_list = final_result['extended_project_list']
     extended_project_list.to_csv(extended_project_list_path, index=False)
-
+    validation_enabled = True
```
Comment (akey7) [inline main.py:51]: Is there a reason validation_enabled is being force to `True`? that should be set with the command line option `--validate`

**Comment by parangat94** [comment]
@akey7 merged `develop` into `branch issue_108_integrate_DW` followed by merge of `issue_108_integrate_DW` into current branch (`issue_111_integrate_DW_Erection`).

PR ready for final review.

**Pull request merged**